KoikatuGameplayMod/KoikatuGameplayMod.cs

```diff
@@ -14,13 +14,15 @@ namespace KoikatuGameplayMod
     [BepInPlugin(GUID, "Koikatu Gameplay Tweaks and Improvements", Version)]
     [BepInProcess(GameProcessName)]
     [BepInProcess(GameProcessNameSteam)]
+    [BepInProcess(GameProcessNameVR)]
```

Comment:
Can you also make it support the Steam version of the VR module?

Reply:
Unfortunately I don't have the steam version of VR so I won't be able to test it.
There has been one report of my own VR plugins not working in the steam version when they work fine in the JP version.

Reply:
I see, in that case I will merge the PR and test it in the steam version later.